src/styles/blocks/card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+}
+
+.card__image {
+  margin: 0 auto 40px;
+  display: block;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  margin-bottom: 4px;
+  color: #060b35;
+}
+
+.card__code {
+  font-size: 10px;
+  line-height: 14px;
+  font-style: normal;
+  color: #616070;
+  margin-bottom: 16px;
+}
+
+.card__rating {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 19px;
+  margin-bottom: 24px;
+}
+
+.card__reveiws-counter {
+  font-size: 10px;
+  color: #060b35;
+}
+
+.card__price {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  gap: 60px;
+  margin-bottom: 16px;
+}
+
+.card__price-word {
+  font-size: 12px;
+  color: #616070;
+  text-align: center;
+}
+
+.card__price-sum {
+  font-weight: 700;
+  color: #060b35;
+  font-size: 16px;
+  font-style: bold;
+  text-align: center;
+}
+
+.card__button {
+  display: block;
+  padding: 11px 0;
+  height: 40px;
+  text-decoration: none;
+  color: white;
+  background-color: #00acdc;
+  font-size: 14px;
+  line-height: 16px;
+  border-radius: 5px;
+  text-align: center;
+  text-transform: uppercase;
+
+}
+
+.card__button:hover {
+  border: 1px solid #00acdc;
+  border-radius: 5px;
+  color: #00acdc;
+  background-color: white;
+}
```

Comment:
here it is better to use `.card:hover .card__button{...}` because the card must react immediately to the card itself and not only to the button

Reply:
you are right! Thank you for your review!